docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/account-management-statements/built-in-roles.md

```diff
@@ -5,6 +5,8 @@
 - God
   - 初始 Root 用户，与 Linux 系统中的 Root 用户类似。
```

Comment:
Don't introduce the concept of `root`. God is enough. 

Too hard to comprehend two similar concepts.